.ci/jobs/e2e-testing-mbp.yml

```diff
@@ -3,7 +3,7 @@
     name: stack/e2e-testing-mbp
```

Comment:
Review the line https://github.com/elastic/e2e-testing/pull/156/files#diff-4e9b5ec7241fc139562d9d01515eea2eL28 to avoid the evil indexing

Reply:
What value do you suggest?

Reply:
`ignore-target-only-changes: true`